src/measure_theory/set_integral.lean

```diff
@@ -38,7 +38,7 @@ theorem for a locally finite measure `μ` and a function `f` continuous at a poi
 
 ## Notation
 
-`∫ a in s, f a` is `measure_theory.integral (s.indicator f)`
+`∫ a in s, f a ∂μ` is `measure_theory.integral (μ.restrict s) f`
```

Comment:
Please reference both valid notations (`∫ a in s, f a` and `∫ a in s, f a ∂μ`), and note that they are introduced in the earlier file `measure_theory.bochner_integration` (not this file).